### ISSUE TITLE
Registration marks - working area not correct on Cameo 2

### ISSUE THREAD

**Issue by bianchifan** [jnweiger/inkscape-silhouette#205]
1st...
- Ubuntu Studio 22.04, Inkscape 1.1
- Linix Mint 20.3, Inkscape 1.2

Regmarks work perfectly with latest plugin :))

But the cutting area  shown by the actual regmarks SVG is not 100% correct.
Top, right and bottom borders must be placed exactly 10mm from the edges and the left border 20mm from the left edge.

Current registration-marks-cameo-silhouette-a4.svg
![registration-marks-cameo-silhouette-a4 svg](https://user-images.githubusercontent.com/17491743/184634052-5e7e5ae3-8b95-45ad-8fdc-189418e0a3e9.png)

Correct version for Silhoutte Cameo 2: Registration Marks Cameo2.svg
![Registration Marks Cameo2 svg](https://user-images.githubusercontent.com/17491743/184634287-7a4afaa0-9748-4d99-9ff5-b311214c22b7.png)
[Registration Marks Cameo2.svg.zip](https://github.com/fablabnbg/inkscape-silhouette/files/9337130/Registration.Marks.Cameo2.svg.zip)

**Comment by EtherGraf** [comment]
Yeah, I think you have found a bug with the internal bounding box.

I think I have seen some similar effects some years ago. 
As far as i remember, the internal bounding box is not corrected by the 10mm offset of the regmarks. This leads to an double sized uncut area on the left side. I would expect the same beeing the case on the top border.

**Comment by bianchifan** [comment]
> This leads to an double sized uncut area on the left side. 

Yeah, that's correct..
On all other sides the uncut area is very small, ~1.3 mm.
The border layson top of the regmarks, not on their center lines.
If you place your design exactly at the border the outer lines are not cut
![9 7mm Borderline](https://user-images.githubusercontent.com/17491743/184883172-6fb347d9-426f-4769-b236-4a016cb51a52.jpg)
.

**Comment by mofosyne** [comment]
Is this still an issue now? Especially now that we can render our own regmark?
